apps/design-system/content/docs/ui-patterns/form-patterns.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Form patterns
```

Comment:
Could just be titled _Forms_ to match other contents of _UI Patterns_ contents?

<img width="50%" height="298" alt="Image" src="https://github.com/user-attachments/assets/dda69a39-4699-4aae-a226-f4138df29939" />